mesh/generator/generator.go

```diff
@@ -133,6 +133,7 @@ func BuildMeshMap(ctx context.Context, o mesh.Options, gi *mesh.AppenderGlobalIn
 
 		infraData := map[string]any{
 			"config":     cp.Config,
+			"configMap":  cp.ConfigMap,
```

Comment:
Same question here about `config` no longer being used and therefore not needing to send it to the frontend?